geemap/cartoee.py

```diff
@@ -1023,7 +1023,10 @@ def add_legend(ax, legend_elements=None, loc="lower right"):
         ax (cartopy.mpl.geoaxes.GeoAxesSubplot | cartopy.mpl.geoaxes.GeoAxes): required cartopy GeoAxesSubplot object.
         legend_elements (list, optional): A list of legend elements. Defaults to None.
         loc (str, optional): Location of the legend, can be any of ['upper left', 'upper right', 'lower left', 'lower right']. Defaults to "lower right".
-
+        font_size(int|string, optional): Font size. Either an absolute font size or an relative value of 'xx-small', 'x-small', 'small', 'medium', 'large', 'x-large', 'xx-large'. defaults to 14.
+        font_weight(string|int, optional): Font weight. A numeric value in the range 0-1000 or one of 'ultralight', 'light', 'normal' (default), 'regular', 'book', 'medium', 'roman', 'semibold', 'demibold', 'demi', 'bold', 'heavy', 'extra bold', 'black'. Defaults to 'normal'.
+        font_color(str, optional): Text color. Defaults to "black".
+        show_chinese(bool, optional): Whether to display Chinese fonts. If True, using 'SimHei' font.
```

Comment:
font_family (str, optional): The font family to use. It supports Chinese fonts, e.g., `SimHei`.